packages/backend-test-utils/src/database/TestDatabases.test.ts

```diff
@@ -184,15 +184,15 @@ describe('TestDatabases', () => {
     );
 
     itIfDocker(
-      'obeys a provided connection string for postgres 9',
+      'obeys a provided connection string for postgres 12',
       async () => {
         const { host, port, user, password, stop } =
-          await startPostgresContainer('postgres:9');
+          await startPostgresContainer('postgres:12');
 
         try {
           // Leave a mark
-          process.env.BACKSTAGE_TEST_DATABASE_POSTGRES9_CONNECTION_STRING = `postgresql://${user}:${password}@${host}:${port}`;
-          const input = await dbs.init('POSTGRES_9');
+          process.env.BACKSTAGE_TEST_DATABASE_POSTGRES12_CONNECTION_STRING = `postgresql://${user}:${password}@${host}:${port}`;
```

Comment:
should keep the tests for the old ids around (expected to only see entirely new additions in this file, no changes to old code)

Reply:
Fixed up 👍